content/webapp/pages/search/index.tsx

```diff
@@ -183,23 +224,71 @@ const NewSearchPage: NextPageWithLayout<NewProps> = ({
               </div>
             </Container>
           </BasicSection>
-
-          {(catalogueResults.images || catalogueResults.works) && (
-            <div>
-              {catalogueResults.images && (
-                <div>
-                  <SectionTitle sectionName="Images" />
-                  <p>{catalogueResults.images?.totalResults || 0} results</p>
-                </div>
-              )}
-              {catalogueResults.works && (
+          <div>
+            {catalogueResults.works &&
+              catalogueResults.works.totalResults > 0 && (
                 <div>
```

Comment:
worth keeping `{(catalogueResults.images || catalogueResults.works)` to keep an empty div from rendering? Although I think we'll address this when we have designs for no results, so not necessary

Reply:
Oops this is for line 227 🤦‍♀️

Reply:
yeah, I'll take care of this in the styling work